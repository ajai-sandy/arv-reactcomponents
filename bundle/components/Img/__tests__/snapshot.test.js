import React from "react";
import { mount, shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import TestComponent from "../test.component";

describe("Root Component Tests", () => {
  it("Matches Shallow SnapShot", () => {
    expect(shallowToJson(shallow(<TestComponent />))).toMatchSnapshot();
  });

  it("Matches Mounted SnapShot", done => {
    const mounted = mount(<TestComponent />);

    setTimeout(() => {
      expect(mounted).toMatchSnapshot();
      done();
    }, 300);
  });
});
