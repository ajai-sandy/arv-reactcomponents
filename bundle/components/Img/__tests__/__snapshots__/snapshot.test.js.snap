// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Component Tests Matches Mounted SnapShot 1`] = `
<ImgTest>
  <Img
    alt="NNNOW"
    className=""
    onError={[Function]}
    onLoad={[Function]}
    src="https://static.nnnow.com/client/assets/images/icon/defaultSkuImage.png"
  >
    <img
      alt="NNNOW"
      className="nwc-img "
      onError={[Function]}
      onLoad={[Function]}
      src="https://static.nnnow.com/client/assets/images/icon/defaultSkuImage.png"
    />
  </Img>
</ImgTest>
`;

exports[`Root Component Tests Matches Shallow SnapShot 1`] = `
<Img
  alt="NNNOW"
  className=""
  onError={[Function]}
  onLoad={[Function]}
  src="https://static.nnnow.com/client/assets/images/icon/defaultSkuImage.png"
/>
`;
